### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jordiubanell**

### CODE REVIEW

File: staff/jordi-ubanell/safe-box/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
 </head>
 <body>
     <script src="safe-box.js"></script>
+<<<<<<< HEAD
     <!-- <script src="safe-box.test.js"></script> -->
```
Comment (pausanchez) [inline staff/jordi-ubanell/safe-box/index.html]: Commented line - when the code is above repeated

File: staff/jordi-ubanell/spotify-app/scripts/logic.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+var logic = {
```
Comment (SergioCastilloSorribes) [inline staff/jordi-ubanell/spotify-app/scripts/logic.js]: Falta Use Strict!!!!

Comment (alejaa03) [reply staff/jordi-ubanell/spotify-app/scripts/logic.js]: WTF USE STRICT??!??!

Comment (jordiubanell) [reply staff/jordi-ubanell/spotify-app/scripts/logic.js]: it's ok!

File: staff/jordi-ubanell/safe-box/index.html (hunk 1 above)
Comment (jordiubanell) [reply staff/jordi-ubanell/safe-box/index.html]: ready!

**Pull request merged**